### PR TITLE
chore(post): fix minor typo

### PR DESCRIPTION
**Pull request by wKovacs64**

### CODE REVIEW

File: content/blog/using-fetch-with-type-script/index.mdx [hunk 1]
```diff
@@ -309,7 +309,7 @@ In case you're curious, here's the type definition for `Object.assign`:
 assign<T, U>(target: T, source: U): T & U;
 ```
 
-And that's it! We've no successfully typed `fetch` for a particular request. 🎉
+And that's it! We've now successfully typed `fetch` for a particular request. 🎉
```
Comment (willian) [inline content/blog/using-fetch-with-type-script/index.mdx:312]: Cool! I was about to open a PR with this fix, glad that I checked PRs before it :)

**Comment by JacobMGEvans** [comment]
@all-contributors please add @arvigeus for content
add @willian for review

**Comment by JacobMGEvans** [comment]
@all-contributors please add @arvigeus for content

**Comment by JacobMGEvans** [comment]
@all-contributors add @willian for reviewing

**Comment by JacobMGEvans** [comment]
@all-contributors add @willian for review

**Review by JacobMGEvans — APPROVED** [review]

**Comment by wKovacs64** [comment]
@all-contributors please add @wKovacs64 for content

**Pull request merged**